spec/factories/user.rb

```diff
@@ -6,4 +6,13 @@
   password_confirmation "password"
   confirmed_at Time.now
  end
+
```

Comment:
Indentation could do with a little tidying in this file. Ruby convention is 2 spaces per indent.